index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>José Fernández-Caminero: Full-Stack Engineer; Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <nav>
+        <img class="artdeco1" src="resources/image_geometry_1.svg" alt="Geometrical decoration">
+        <img src="resources/ic_menu.svg" alt="Toolbar menu">
+    </nav>
```

Comment:
- [ ] please it would be better if you put the `<nav>` tag inside `<header>`, always try to use the right semantic tags  as much as possible